planner/core/common_plans.go

```diff
@@ -404,13 +404,15 @@ type AnalyzeColumnsTask struct {
 	PhysicalTableID int64
 	PKInfo          *model.ColumnInfo
 	ColsInfo        []*model.ColumnInfo
+	Table           table.Table
```

Comment:
Why we need it?

Reply:
We need the table info to build the range key for each region in fast analyze.

Reply:
We already have table id, and all the column/index info, what else do we need?

Reply:
When building index value in the fast analyze. We need the columns in the table with keeping order.